prow/config/get_labels.js

```diff
@@ -0,0 +1,28 @@
+function rgbToHex(col) {
```

Comment:
Actually, we have label dumpling tool: https://github.com/ti-community-infra/tichi/tree/master/tools/label-dumpling